Makefile

```diff
@@ -69,7 +71,8 @@ CONFIG_AP_WOWLAN = n
 ######### Notify SDIO Host Keep Power During Syspend ##########
 CONFIG_RTW_SDIO_PM_KEEP_POWER = y
 ###################### Platform Related #######################
-CONFIG_PLATFORM_I386_PC = y
+CONFIG_PLATFORM_ARM_JET_NANO = y
+CONFIG_PLATFORM_I386_PC = n
```

Comment:
Can you maybe leave this to yes by default ?
And put your platform to `n`

Reply:
@lion24 done. you can review and merge.